extensions/ohif-i18n/src/locales/en-US/index.js

```diff
@@ -3,7 +3,6 @@ import CineDialog from './CineDialog.json';
 import Common from './Common.json';
 import Header from './Header.json';
 import MeasurementTable from './MeasurementTable.json';
-import UserPreferencesModal from './UserPreferencesModal.json';
```

Comment:
There shouldn't be any changes to the JSON or index.js files.

Reply:
done